src/Symfony/Component/Intl/Resources/emoji/build.php

```diff
@@ -143,20 +143,29 @@ public static function buildSlackRules(array $emojisCodePoints): iterable
         foreach ($emojis as $data) {
             $emojiCodePoints = str_replace('-', ' ', strtolower($data['unified']));
             $shortCode = $data['short_name'];
+            $shortCodes = $data['short_names'];
+            array_walk($shortCodes, function (&$value, $key) {
```

Comment:
I'd rather use `array_map` than `array_walk` that changes the array by reference. It makes the code easier to read.